src/test/java/com/android/tools/build/bundletool/commands/BuildBundleCommandTest.java

```diff
@@ -233,6 +233,24 @@ public void buildingViaFlagsAndBuilderHasSameResult_optionalUncompressed() throw
     assertThat(commandViaBuilder).isEqualTo(commandViaFlags);
   }
 
+  @Test
```

Comment:
I think we also need a test that actually tests the feature, i.e. that verifies that the command succeeds when the file already exists and that the old file has been replaced.

Reply:
Would you like me to add a test for this for the option in BuildApksCommand? I couldn't find an existing test for that functionality.

Reply:
Oh yes, that would be good, thanks! Either in same PR or separate one, up to you.